fastlane/lib/assets/ActionDetails.md.erb

```diff
@@ -50,12 +50,37 @@ Key | Description | Default
 <% end %><%# End of action.available_options... %>
 
 <hr />
+
+## Documentation
+
 To show the documentation in your terminal, run
 ```no-highlight
 fastlane action <%= action.action_name %>
 ```
 
-<a href="https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/<%= @action_filename %>" target="_blank">View source code</a>
+<hr />
+
+## CLI
+
+It is recommended to add the above action into your `Fastfile`, however sometimes you might want to run one-offs. To do so, you can run the following command from your terminal
+
+```no-highlight
+fastlane run <%= @action.action_name %>
+```
+
+To pass parameters, make use of the `:` symbol, for example
+
+```no-highlight
+fastlane run <%= @action.action_name %> parameter1:"value1" parameter2:"value2"
+```
+
+It's important to note that the CLI doesn't support setting all kinds of parameters (e.g. Arrays are not supported). It is recommended to add all _fastlane_ actions you use to your `Fastfile`.
```

Comment:
@KrauseFx Arrays are actually supported if they are defined with `type: Array` through a comma delimited string (ie: `value:"1,2,3"` will get parsed as `["1","2","3"]`). `type: Hash` is currently not supported via CLI though.

Reply:
Can you update the PR directly so that the phrasing is correct?